Sources/PackageGraph/RepositoryPackageContainer.swift

```diff
@@ -124,6 +104,28 @@ public class RepositoryPackageContainer: BasePackageContainer, CustomStringConve
             currentToolsVersion: currentToolsVersion
         )
     }
+    
+    // Compute the map of known versions.
+    private func knownVersions() throws -> [Version: String] {
+        try self.knownVersionsCache.memoize() {
+            let knownVersionsWithDuplicates = Git.convertTagsToVersionMap(try repository.tags())
+
+            return knownVersionsWithDuplicates.mapValues({ tags -> String in
+                if tags.count == 2 {
```

Comment:
Unrelated to the PR since this is existing logic, but is `== 2` ok or should this be `> 1` (if possible)?

Reply:
yes, this also caught my eye but wanted to minimize changes to existing logic. will create a follow-on PR?